File: python/Day02.py
#Given the meal price (base cost of a meal), tip percent (the percentage of the
#meal price being added as tip), and tax percent (the percentage of the meal
#price being added as tax) for a meal, find and print the meal's total cost.


meal = float(raw_input())
tip = float(raw_input())
tax = float(raw_input())

tip = meal*(tip/100)
tax = meal*(tax/100)
cost = meal+tip+tax
if cost>(int(cost)+0.5) :
    cost = str(int(cost)+1)
else :
    cost = str(int(cost))

print "The total meal cost is "+cost+" dollars."

File: C++/Day02.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    float cost,totalCost,tipPercent,taxPercent,tip,tax;
    cin >> cost >> tipPercent >> taxPercent;

    tip = (tipPercent/100)*cost;
    tax = (taxPercent/100)*cost;

    totalCost = tip+tax+cost;

    if (totalCost > int(totalCost) + 0.5) {
        totalCost=int(totalCost)+1;
    }
    else {
        totalCost = int(totalCost);
    }

    cout << "The total meal cost is " << totalCost << " dollars.";
    return 0;
}
